### PR TITLE
Fix linter issues

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Fixes linter issues in tests.

## Testing

1. `gulp lint` should have no errors.
2. `gulp test:unit` should pass.

### CODE REVIEW

File: test/unit-test/src/cf-atomic-component/src/utilities/on-ready-spec.js [hunk 1]
```diff
@@ -44,14 +45,12 @@ describe( 'on-ready', function() {
 
   it( 'should add a function to the saved array and trigger it' +
       'when readyState completes', () => {
-    let readyReturn;
```
Comment (anselmbradford) [inline test/unit-test/src/cf-atomic-component/src/utilities/on-ready-spec.js]: @contolini this is unused, right?

**Comment by jimmynotjim** [comment]
@anselmbradford Are you getting errors for lcov-report files? Not sure why they'd be captured

**Comment by anselmbradford** [comment]
> Are you getting errors for lcov-report files? Not sure why they'd be captured

@jimmynotjim Hm, no I'm not. Just ran `gulp lint && gulp test:unit` and I don't see any lcov anything.

**Comment by jimmynotjim** [comment]
Doh, nvm, I never deleted the old build-coverage directory. All is well

**Pull request merged**